tasks/sep/006-1-sc-changes/VALIDATION.md

```diff
@@ -37,20 +37,18 @@ Links:
 **Meaning:** The threshold has been increased to 3. The key can be validated by the location of the `threshold` variable in the [Safe's Storage Layout](https://github.com/safe-global/safe-smart-account/blob/186a21a74b327f17fc41217a927dea7064f74604/contracts/examples/libraries/GnosisSafeStorage.sol#L14). The before value shown is 1, rather than the actual value of 2, because of the state override.
```

Comment:
_(arbitrary comment location to start a thread)_

A few addresses in this file have `0x0x` instead of `0x`, including some URLs

Reply:
fixed